requirements.txt

```diff
@@ -0,0 +1,4 @@
+azure-datalake-store>=0.0.46,<0.1
```

Comment:
Is pinning to `<.1` necessary? I'm not familiar with azure's version policy.

Reply:
Probably not.  I don't think they will do much additional development on the azure-datalake-store library, since its for the Gen1 datalake.  However, be aware that they're actively working on the storage library, so it should be pinned to <3.0